debug/index.js

```diff
@@ -0,0 +1,3 @@
+if (process.env.NODE_ENV!=='production') {
```

Comment:
Should we remove the `env` check in `src/index.js` in favor of this one?

Reply:
Might also fix the UMD build of `preact/debug` which has a `'development' === process.env.NODE_ENV` check in it which would fail in the browser

Reply:
Yes probably